quickwit-core/src/indexing/split.rs

```diff
@@ -296,8 +288,8 @@ mod tests {
             },
         );
         mock_metastore
```

Comment:
you can remove these 4 lines entirely if you expect it 0 times. If it is called the unit test will panic.